docs/howto/index.rst

```diff
@@ -15,3 +15,4 @@ How-to guides
    use-extensions/index
    /common/craft-parts/how-to/include_files
    /common/craft-parts/how-to/override_build
+   /common/craft-parts/how-to/use_parts
```

Comment:
when I had the similar task in Rockcraft, Michael and I decided against adding this file; [source](https://github.com/canonical/rockcraft/pull/838#discussion_r2005370797)

Reply:
Thanks for the reference, removed.